new-lamassu-admin/src/pages/Blacklist/Blacklist.styles.js

```diff
@@ -0,0 +1,42 @@
+export default {
+  grid: {
+    flex: 1,
+    height: '100%'
+  },
+  content: {
+    display: 'flex',
+    flexDirection: 'column',
+    flex: 1,
+    marginLeft: 48
```

Comment:
We're not really using this everywhere but it's good practice to use our `spacer` variable. You can import it from 'src/styling/variables'. 

Then you can make it spacer * 6 instead of 48.